api/src/main/java/org/openmrs/module/stockmanagement/StockManagementActivator.java

```diff
@@ -34,9 +34,9 @@ public void started() {
 		try{
 			List<StartupTask> startUpTasks = new ArrayList<>();
 			startUpTasks.add(new LocationTagsSynchronize());
-			startUpTasks.add(new LocationTreeSynchronize());
+			// startUpTasks.add(new LocationTreeSynchronize());
```

Comment:
If there is anything we can do to keep this but only update where necessary the better.

Reply:
He we can use global properties to condiitonaly run this